### ISSUE TITLE
Iodination on tyrosine as labile modification

### ISSUE THREAD

**Issue by lgamon** [Nesvilab/FragPipe#574]
Hi there,

I've been working to identify iodinated peptides(Y+125.8966) in complex samples (using both Bruker QTOF and TimsTOF data). MsFragger does generally a good job of identifying iodotyrosine modifications but we have seen in manual inspection of the MSMS spectra that there is frequently a neutral loss of 125.8966 that occurs, especially in the y-series. I would like to improve the number of PSM matches by including this as a labile modification.

Attempts so far actually yield fewer PSMs than a standard open or closed search. I suspect that I am not setting up the labile search correctly. I've tried simply modifying the labile phospho workflow to a variable mod mass of 125.8966 on Y and an offset of 0/125.8966 (with or without iodotyrosine as a variable mod). No luck so far.

Do you have any advice for setting up this kind of search?

[fragger.params.zip](https://github.com/Nesvilab/FragPipe/files/7821295/fragger.params.zip)

**Comment by dpolasky** [comment]
Hi @lgamon,
Nothing jumps out at me from your parameter file as an obvious problem, but it's unusual for the combined variable mod/offset search to return fewer PSMs than the equivalent closed (variable mod only) search if the modification is at all labile. Could you share the closed search parameters you used as well to compare? 
- Dan

**Comment by fcyu** [comment]
Hi @lgamon ,

Can you send us the log files from the open, closed, and the labile search?

Thanks,

Fengchao

**Comment by lgamon** [comment]
Hi Dan and Fengchao,

Here are the labile/closed searches log files:

[log_2022-01-04_11-06-05.txt](https://github.com/Nesvilab/FragPipe/files/7841292/log_2022-01-04_11-06-05.txt)
[log_2022-01-03_17-25-49.txt](https://github.com/Nesvilab/FragPipe/files/7841308/log_2022-01-03_17-25-49.txt)

**Comment by dpolasky** [comment]
Hi @lgamon,
It looks like the difference in PSMs between the two searches is probably caused by using Percolator (on each file individually) for the closed search but PeptideProphet (combined on all files) for the labile search. I think if you use the same tool for post-processing, the results will be easier to interpret. We don't usually recommend Percolator for mass offset (labile) mode searches because it doesn't take the mass offset modification into account, but it does work. Probably not ideal though unless the modification is quite common (so the probability of finding a modified vs unmodified peptide is similar). So I think switching the closed search to PeptideProphet would be best (though switching the labile search to Percolator is possible), and then the comparison should make more sense.
Best,
Dan

**Comment by fcyu** [comment]
> Hi @lgamon, It looks like the difference in PSMs between the two searches is probably caused by using Percolator (on each file individually) for the closed search but PeptideProphet (combined on all files) for the labile search. I think if you use the same tool for post-processing, the results will be easier to interpret. We don't usually recommend Percolator for mass offset (labile) mode searches because it doesn't take the mass offset modification into account, but it does work. Probably not ideal though unless the modification is quite common (so the probability of finding a modified vs unmodified peptide is similar). So I think switching the closed search to PeptideProphet would be best (though switching the labile search to Percolator is possible), and then the comparison should make more sense. Best, Dan

Hi @dpolasky ,

I am sorry that I haven't read the log files, but does it mean that `closed search+Percolator` gives more IDs than `labile search+PeptideProphet`?

Best,

Fengchao

**Comment by dpolasky** [comment]
@fcyu yes, more PSMs overall, but I'm not sure if it's more modified PSMs or not (can't tell from the logs). It could be that Percolator is finding more unmodified PSMs, but the labile search is finding more modified PSMs despite slightly lower overall 'sensitivity'

**Comment by lgamon** [comment]
Thanks for the help @dpolasky and @fcyu. I've replicated this with a search on iodinated bovine serum albumin (we have validated a lot of the spectra for this dataset) and see exactly what you mention. Only 2 iodination PSMs for a closed+peptideprophet search compared to 43 with mass-offset (a la labile phospho) and a whopping 274/339 for closed/open with percolator.

So yes - when comparing using peptideprophet the mass-offset search does a much better job. Can you suggest why closed+percolator gives so many more PSMs for the modification? They give a very similar number of overall PSMs (closed+percolator = 2916, closed+peptideprophet = 3158, labilesearch+peptideprophet = 2917).

The real dataset I would like to search has a very low occurrence of the modification so I'm trying to optimise the search to give it the best chance of finding any iodinations. It sounds like you would advise against using percolator in this circumstance though.

**Comment by lgamon** [comment]
Oops - I was running the peptideprophet searches with PTMprophet for the phospho mod instead of with --mods Y:125.8966.

After correcting this I now get 303 PSMs with labilesearch+peptideprophet(+PTMprophet --mods Y:125.8966) which is relatively close to the standard open+percolator search.

**Comment by dpolasky** [comment]
Thanks for doing the testing - that's quite interesting that Percolator gets so many modified PSMs from a closed search, but good that the labile search + PeptideProphet is close. We have seen some cases where Percolator boosts rare modifications too much (giving a lot of false hits) in the past, so I'm a little worried that might be happening here with the closed search results (since it should be pretty hard to find very labile modifications with a closed search in general). Do you have a sense of whether the modified PSMs from the Percolator search are mostly the same or different from those found in the labile/PepProphet search? 

If you're able to share the raw data we'd love to take a look at testing the Percolator + labile search internally to see if we can add that as an option.
